.github/workflows/icu_envtest.yml

```diff
@@ -0,0 +1,147 @@
+# Copyright (C) 2023 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# GitHub Action configuration script for ICU environment test tasks.
+#
+
+name: GHA EnvTest
+
+on:
```

Comment:
Yes, this should be enough for manual trigger

Reply:
how to trigger manually?